docs/developers/developing.rst

```diff
@@ -109,21 +109,14 @@ Hypothesis uses Karma and mocha for testing. To run all the tests once, run:
 
    make test
 
-You can filter the tests which are run by running ``make test ARGS='--grep <pattern>'``,
-or ``yarn test --grep <pattern>``. Only test files matching the regex ``<pattern>`` will
-be executed.
```

Comment:
I removed the `make` variant here not because it's not _true_ (we can add it back if so desired) but just because I'm trying to minimize complexity. The convention I'm going for here is to lean toward `make` for full runs of things (e.g. `make test`, `make sure`) and `yarn` for filtered runs. We can discuss this and tweak as desired.

Reply:
That's in fact how I usually do it. It seems like a sensible and intuitive approach.